src/main/java/models/Employee.java

```diff
@@ -0,0 +1,52 @@
+package models;
+
+import java.sql.Timestamp;
+
+public class Employee {
+
+  private long employeeId;
+  private String firstName;
+  private String lastName;
+  private Timestamp addedDate;
+  private long status;
```

Comment:
This should change, and even if it was a status your value would be at most 1 digit long or two, so `int` is more appropriate or even `boolean`. It should have meaning with the db table. You should assume you know what the data will be before creating the structure.